CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Remove deprecated platform in spec tests
 
+## 6.2.2
```

Comment:
```suggestion
## Unreleased
```

Reply:
oh do you want these changes to go into Unreleased?

Reply:
@ramereth - removed 6.2.2 and added into unreleased